src/components/Grid/Grid.css

```diff
@@ -0,0 +1,270 @@
+ * 
+ * This specifies what column a cell should start at. 
+ * For example you can start a cell at column 6 (.cell-start-N) and have 
+ * it span 6 columns (.cell-6) making it go from the middle to the right 
+ * edge of the grid. 
+ *
+ * SHORTHAND: .cell-<start>-<span>
+ * ex. .cell-3-6 = start at cell 3 and span 6 cells 
+ *
+ * RESPONSIVE: .sm:cell-N-N .sm:cell-start-N
+ * corresponding css breakpoint classes (.sm:, .md:, .lg:) are available
+ * to change the cell width at specified breakpoints giving you tons of flexibilit
+   ========================================================================== */
+   @variants responsive {
+
+    @for $i from 1 to 12 {
+      @for $x from 11 to 1 {
+        .cell-$i.cell-start-$x {
+          grid-column: $(x) / span $(i);
+    
+          min-width: 0;
+        }
+        /* shorthand cell-<start>-<span> */
+        .cell-$(x)-$(i) {
+          grid-column: $(x) / span $(i);
+          min-width: 0;
+        }
+      }
+    }
+    
+  }
+
+/* Tablet
+  ========================================================================== */
+
+
+
+/* =======================================================================
+  IE 11 css grid fallback
+
+  This media query is hack to only target IE11 properties.
+  Sine IE 11 does NOT support grid gap property we create a 24 column 
+  grid with alternating "gap" columns
+======================================================================= */
+
+@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+  .grid-container {
+    border: 1px solid red !important;
+    /* autoprefixer grid: autoplace */
+    max-width: 960px;
+
+    padding: 0 0 0 20px;
+    display: grid;
+    align-items: center;
+    flex-wrap: wrap;
+
+    /* 
+    * 24 columns (2 columns repeated 12 times):
+    * odd numbered columns each taking up the available space 1fr (fraction unit) 
+    * even numbered columns are "gap" columns which are 20px wide
+    * */
+
+    grid-template-columns: repeat(12, 1fr 20px);
+
+    &--collapsed {
+      /*
+       * collpase our container padding and the width of the even columns
+       * so we keep the gap columns and the .cell-N propery math still works
+       */
+      padding: 0;
+      grid-template-columns: repeat(12, 1fr 0px);
+      [class*='row-'] {
+        margin: 0;
+      }
+      /* collapse horizontal guttes only */
+      &-cells {
+        padding: 0;
+        grid-template-columns: repeat(12, 1fr 0px);
+      }
+
+      /* collapse vertical guttes only */
+      &-rows {
+        [class*='row-'] {
+          margin: 0;
+        }
+      }
+    }
+
+    /* make full width of container */
+    &--fullWidth {
+      max-width: 100%;
+    }
+  }
+  .grid-container.grid-container--full-width {
+    max-width: 100%;
+  }
+
+  /* 
+ * Rows .row-N - unlimited number
+ * 
+ * add margin to fake the grid gap
+========================================================================== */
+
+  @for $i from 1 to 12 {
```

Comment:
Why is this one only 12 and the non-ie was 100?

Reply:
good catch, will fix